requests/requests.go

```diff
@@ -383,8 +385,8 @@ func (reqs *ReqsCollection) AcquireWorker(reqID boruta.ReqID) (boruta.AccessInfo
 // the request has acquired worker that to extend time for which the worker is
 // assigned to the request.
 func (reqs *ReqsCollection) ProlongAccess(reqID boruta.ReqID) error {
-	reqs.mutex.RLock()
-	defer reqs.mutex.RUnlock()
+	reqs.mutex.Lock()
```

Comment:
why limit read access for other goroutines?

Reply:
Because in ProlongAccess Job is modified.

Reply:
I think that was done rather by mistake than on purpose.